src/main/cpp/src/CastStringJni.cpp

```diff
@@ -109,6 +109,21 @@ JNIEXPORT jlong JNICALL Java_com_nvidia_spark_rapids_jni_CastStrings_toFloat(
   CATCH_CAST_EXCEPTION(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_com_nvidia_spark_rapids_jni_CastStrings_fromFloat(
+  JNIEnv* env, jclass, jlong input_column, jint j_dtype)
+{
+  JNI_NULL_CHECK(env, input_column, "input column is null", 0);
+
+  try {
+    cudf::jni::auto_set_device(env);
+
+    cudf::column_view cv{*reinterpret_cast<cudf::column_view const*>(input_column)};
```

Comment:
This will call a copy constructor. So let's do something else:
```suggestion
    auto const& cv = *reinterpret_cast<cudf::column_view const*>(input_column);
```